src/askContainer/webViews/deploySkillWebview/deployHostedSkillWebview.ts

```diff
@@ -141,9 +140,9 @@ export class DeployHostedSkillWebview extends AbstractWebView {
                 } else {
                     await deployHostedSkillManager.deploySkill(this);
                 }
-                void telemetryClient.sendData();
+                await TelemetryClient.getInstance().store(action);
```

Comment:
If the `await` is only for storing the data, that is alright. But if is for `store` and `sendData`, then it might lead to performance issues. Any issue with removing `await` here?

Reply:
There is a race condition concern, it may happen rarely.  However, to avoid blocking the UI, removed the `await`.